packages/amplify-util-mock/package.json

```diff
@@ -32,10 +32,10 @@
     "@hapi/topo": "^5.0.0",
     "amplify-appsync-simulator": "^2.3.12",
     "amplify-category-api-dynamodb-simulator": "2.4.3",
-    "amplify-category-function": "^4.0.12",
+    "amplify-category-function": "4.2.0-cdkv2.0",
```

Comment:
This will need caret version back when we release GA. https://app.asana.com/0/1202389680978480/1203201977550367/f